entgql/internal/todofed/ent/entc.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019-present Facebook Inc. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+//go:build ignore
+// +build ignore
+
+package main
+
+import (
+	"log"
+
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent/entc"
+	"entgo.io/ent/entc/gen"
+)
+
+func main() {
+	ex, err := entgql.NewExtension(
+		entgql.WithWhereFilters(true),
+		entgql.WithSchemaPath("../ent.graphql"),
+		entgql.WithConfigPath("../gqlgen.yml"),
```

Comment:
The example looks good, but don't you think we should add to this an example for passing [`api.Option`](https://github.com/99designs/gqlgen/blob/master/api/option.go#L8)s?

Reply:
There aren't have many options to demo. Built in options are NoPlugins and WithPlugIn that require some thirdpary implement. I think its complicated to demo.